Source/Core/CesiumTerrainProvider.js

```diff
@@ -1030,7 +1027,7 @@ define([
         var availabilityTilesLoaded = layer.availabilityTilesLoaded;
         var availability = layer.availability;
 
-        var tile = getAvailabilityTile(layer, x, y, level);
+        var tile = CesiumTerrainProvider._getAvailabilityTile(layer, x, y, level);
```

Comment:
Bit of a nitpick, but why not just call `getAvailabilityTIle` instead of `CesiumTerrainProvider._getAvailabilityTile`? The former should be a tiny bit faster.

Alternatively, define this function as `CesiumTerrainProvider._getAvailabilityTile = function(layer, x, y, level) { ... };` in the first place and make the source code a tiny bit smaller.